conf/api.yml

```diff
@@ -984,6 +984,22 @@ paths:
             type: string
       tags:
         - crx
+    /crx/server/crx.default/jcr:root/.1.json:
+      get:
+        operationId: getCrxdeStatus
+        produces:
+          - application/json
+        responses:
+          200:
+            description: 'CRXDE is enabled'
```

Comment:
Please introduce schema here, e.g. `#/definitions/CrxdeStatus` .
Use getInstallStatus as an example.

This definition will then produce an object model, which will then be propagated back to the caller.
The intention of having this model is to save the caller from having to parse JSON payload.